generators/ariane

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env python3
 
```

Comment:
After a bunch of umming and ahhing in the #symbiflow IRC channel we decided that this newline in Python files should have a hash to join the hashbang and the copyright statement.
```suggestion
#
```